libcxx/include/__string/constexpr_c_functions.h

```diff
@@ -123,7 +123,7 @@ __constexpr_memcmp_equal(const _Tp* __lhs, const _Up* __rhs, __element_count __n
     }
     return true;
   } else {
-    return __builtin_memcmp(__lhs, __rhs, __count * sizeof(_Tp)) == 0;
+    return ::__builtin_memcmp(__lhs, __rhs, __count * sizeof(_Tp)) == 0;
```

Comment:
Just curious, but why is this one necessary?

Reply:
Apparently GCC does ADL lookup on builtins.

Reply:
Thanks for explaining. I guess this makes sense, since this could be a function defined somewhere.